client/db_test.go

```diff
@@ -0,0 +1,28 @@
+package client_test
+
+import (
+	"fmt"
+
+	"github.com/cockroachdb/cockroach/client"
+	"github.com/cockroachdb/cockroach/server"
+)
+
+func ExampleDB() {
+	s := server.StartTestServer(nil)
+	defer s.Stop()
+
+	db := client.Open(s.ServingAddr())
+	db.Put("hello", "world")
+	result, _ := db.Get("hello")
+	fmt.Println(result.Rows[0].ValueString())
+
+	db.Inc(100, "world")
```

Comment:
`Inc(value, key)` feels really unnatural, especially in comparison to `Put(key, value)`. I understand that this is the only way we can get variadic increments, but do we really need that?

For that matter I'm not a fan of variadic Put() either, since we have explicit batches. I think it would be cleaner to just call Batch.Put multiple times even if it is more verbose.